index.js

```diff
@@ -65,6 +62,10 @@ taskPlugins
       throw new Error('Plugin named ' + pluginDefinition.name + ' does not export a function');
     }
 
-    var taskDefinition = plugin(pluginContext);
+    var taskDefinition = plugin({
+      taskYargsRun: taskYargsRun,
+      gulp: gulp,
+      runSequence: runSequence
+    });
```

Comment:
plugin registry context, and the context used to configure each plugin, are also now separate